src/Framework/ContaoFramework.php

```diff
@@ -243,10 +243,14 @@ private function getRoute()
             return 'console';
         }
 
-        $route = $this->router->generate(
-            $this->request->attributes->get('_route'),
-            $this->request->attributes->get('_route_params')
-        );
+        try {
+            $route = $this->router->generate(
+                $this->request->attributes->get('_route'),
+                $this->request->attributes->get('_route_params')
+            );
+        } catch (\InvalidArgumentException $e) {
+            return null;
```

Comment:
I agree, but should that be considered a BC break? I very much doubt anyone uses `TL_SCRIPT` on the command line, as it's only necessary for the legacy `initialize.php`…